spark-cassandra-connector/src/test/scala/com/datastax/driver/core/RowMock.scala

```diff
@@ -0,0 +1,114 @@
+package com.datastax.driver.core
```

Comment:
Can't we use ScalaMock for this? It is already on the classpath. In this case reating a mock with just a few methods would be much simpler with ScalaMock.